content/articles/create-an-ajax-type-ahead-feature-for-a-website/index.md

```diff
@@ -0,0 +1,467 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /create-an-ajax-type-ahead-feature-for-a-website/
+title: Create a Type Ahead Feature in JavaScript using Fetch and Regex
+description: This tutorial will discusss how to build a type-ahead feature for a website using Fetch and Regex.
+author: godwin-martins
+date: 2022-02-09T00:00:00-10:50
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/create-an-ajax-type-ahead-feature-for-a-website/hero.jpg
+    alt: Create a Type Ahead Feature in JavaScript using Fetch and Regex Hero Image
+---
+Part of the job of a front-end developer is to create intuitive user interfaces that deliver the best user experience to website users. Part of getting this job done is by creating search suggestions or what is known as the type-ahead feature for websites/webpages.
```

Comment:
I feel the start of your article is not very intuitive, kindly rephrase it. You could start by saying this 
` Creating intuitive user interfaces that deliver the best user experience to website users is part of the job of a front-end developer`.   

Kindly avoid being repetitive - the word `part`.